scripts/stylelint/README.md

```diff
@@ -0,0 +1,91 @@
+## Allowed Custom Properties plugin
```

Comment:
`scripts` is a dumping ground for one-off commands. I reckon this stuff is worth pulling out to a top-level `stylelint` folder (so stylelint/README.md, rather than `scripts/stylelint/README.md`).

Reply:
Sounds good 👍  https://github.com/Shopify/polaris-react/pull/4804/commits/5563716ac291c90666404f8511655965c5f72c75